### ISSUE TITLE
The configuration used for your pretrained weights

### ISSUE THREAD

**Issue by jcliu0428** [zju3dv/NeuralRecon#92]
Hi thanks for sharing your great work!

May I ask a question that what are the configurations you used for the pretrained weights you provided? e.g., the batch size and training epochs, the learning rate and the training split of ScanNet? Since I follow your default setting to train on ScanNet v2 train split but the performance(fscore) has some discrepancy of your pretrained weights, so I wonder whether this is due to different training configuration or split. Looking forward to your reply!
